components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -2573,6 +2579,9 @@ public void updateDocumentation(APIIdentifier apiId, Documentation documentation
             String visibleRolesList = api.getVisibleRoles();
             if (visibleRolesList != null) {
                 authorizedRoles = visibleRolesList.split(",");
+                for(int i = 0; i < authorizedRoles.length; i++) {
```

Comment:
Formatting issues

Reply:
Lets also use enhanced for loop(https://stackoverflow.com/a/18738765/3296350), its a lot cleaner